eCommerce platforms/Adobe Magento/src/code/local/Microsoft/Dfp/Model/Config/Source/AssessmentTypes.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Microsoft Dynamics 365 Fraud Protection module for Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Microsoft
+ * @package    Dynamics 365 Fraud Protection
+ * @copyright  Copyright (c) Microsoft Corporation. (https://www.microsoft.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+class Microsoft_Dfp_Model_Config_Source_AssessmentTypes
+{
+   public function toOptionArray()
+   {
+       return [
+           ['value' => 0, 'label' => __('Evaluate')],
```

Comment:
One thing I forgot to ask before was if this actually changed the behavior of the module or not; for example, if I select Protect, do we actually take the recommendation or is that something that is still manually configured?

Reply:
Further in the review, it looks like it truly does but just want to make sure.

Reply:
Yes, it does probably use it.  Thanks for pointing this out though.  It helped me clean up the coe near where this value is actually read in the OnePageController.